common/table.js

```diff
@@ -1487,7 +1494,18 @@
                         }, scope.vm.reference.defaultLogInfo);
                     });
                 }
-            };
+
+                var overflowContainer = document.getElementById('overflow-container');
+                if (overflowContainer) {
+                    overflowContainer.onscroll =  fixHeader;
+                }
+
+            };  
+
+            var fixHeader = function(e) {
+                var header = document.getElementById('fixed-table-header');
```

Comment:
As mentioned above, change the way you're getting the element. Also comment what this comment is doing.